ddtrace/contrib/pytest/plugin.py

```diff
@@ -130,11 +130,16 @@ def _get_module_path(item):
 
 
 def _get_module_name(item):
-    """Extract module name from a `pytest.Item` instance."""
+    """Extract module name (fully qualified) from a `pytest.Item` instance."""
```

Comment:
Is this `item.module.__name__` ? 🤔 

I was curious and dropped into `pdb` at line `134`
```
(Pdb) item.module.__name__
'test_outer_package.test_inner_package'
```